docs/openapi.yaml

```diff
@@ -824,7 +824,7 @@ components:
         number:
           type: string
           description: Course number.
-          example: 493
+          example: "493"
```

Comment:
In some other change locations, you remove `"`, here you add `"`. Why?

Reply:
Here the course number is set to be a string, e.g. it can be "493", "325", or even "325H" assuming we're replicating OSU's course system.